tests/repo3/data/hafez/divan/hafez.divan.perseus-ger1.xml

```diff
@@ -0,0 +1,120 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Rename the folder to something like `test_auto_rng` and have both remote and local in the same repo